llvm/include/llvm/Transforms/IPO/MergeFunctions.h

```diff
@@ -15,7 +15,9 @@
 #ifndef LLVM_TRANSFORMS_IPO_MERGEFUNCTIONS_H
 #define LLVM_TRANSFORMS_IPO_MERGEFUNCTIONS_H
 
+#include "llvm/IR/Function.h"
```

Comment:
Forward declare this instead.

Reply:
Resolved in [350395d](https://github.com/llvm/llvm-project/pull/111045/commits/350395d7c6e7614e3012443abe155cfc1b307e87).